### DISCUSSION TITLE
[General][Java SDK (Component)] Integrate spi-samples into github-action in the spi repository

### DISCUSSION THREAD

**Discussion by heliang666s** [apache/dubbo#14862] [General]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SPI Extensions (apache/dubbo-spi-extensions)

### Details

Since submitting a PR in the spi library now will only run unit tests,now we need Integrate the spi-samples application library into the github-action of spi-extensions to ensure that the integration test functionality is checked for each CI submission The end result is:.All examples that make each PR commit need to run a sample.

Repository link: https://github.com/apache/dubbo-spi-extensions https://github.com/apache/dubbo-spi-samples

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by shaokeyibb** [comment]
Hello, I want to try this work, can you assign it to me?

**Reply by heliang666s** [reply]
welcome,just go ahead!!!
